packages/utils/spatial-utils/src/spatial.js

```diff
@@ -316,3 +316,39 @@ export async function initializeRasterLayersAndChannels(
   const autoImageLayerDefs = await Promise.all(autoImageLayerDefPromises);
   return [autoImageLayerDefs, nextImageLoaders, nextImageMetaAndLayers];
 }
+
+/**
+ * Convert an array of coordinateTransformations objects to a 16-element
+ * plain JS array using Matrix4 linear algebra transformation functions.
+ * @param {object[]} coordinateTransformations List of objects matching the
+ * OME-NGFF v0.4 coordinateTransformations spec.
+ * @returns {number[]} Plain JS array of 16 numbers representing the Matrix4.
```

Comment:
Can we not return the `Matrix4` class? I thought it inherited from `Array` so should be json-able etc. but maybe not

Reply:
Ah it looks like in v3 of math.gl Array is returned anyway - https://uber-web.github.io/math.gl/docs/upgrade-guide#upgrading-to-v30 will try to upgrade